modules/microshift-backing-up-manually.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-backing-up-manually_{context}"]
+= Backing up {product-title} data manually
+
+If you are not using an OSTree system, you can back up data manually at any time. Data backups are advised prior to any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by running an optional backup command.
+
+.Prerequisites
+* You have root access to the host.
+* {product-title} and related workloads are stopped.
```

Comment:
What's under "related workloads"?
Technically we don't require user to stop Pods (workloads) before backing up

Reply:
I think this is referring to the volume snapshot creation phase of the backup.  It's definitely recommended that users's stop their stateful workloads before creating the snapshot.

Reply:
Yes, I was thinking about that as a best practice: Stop everything before you backup anything.

Reply:
We want admin processes to disrupt workloads as little as possible. Since the `microshift backup` command does not back up user workloads or take snapshots, the user workload pods can stay online.